src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
@Nyholm sorry for having initiated this discussion. In case you are worried that "non-technical" people can't understand the translation, don't worry because the original English message cannot be understood either by those people (`The host could not be resolved.` What the hell is a "host" and who o how "resolves" it?)